kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -2808,6 +2808,13 @@ object KyuubiConf {
       .toSequence(";")
       .createWithDefault(Nil)
 
+  val ENGINE_JDBC_DEFAULT_FETCH_SIZE: ConfigEntry[Int] =
+    buildConf("kyuubi.engine.jdbc.default.fetch.size")
```

Comment:
```suggestion
    buildConf("kyuubi.engine.jdbc.fetch.size")
```

Reply:
I think this PR is purposed for default fetch size, which should be clarified in the PR title and description.